koku/masu/external/kafka_msg_handler.py

```diff
@@ -762,73 +763,56 @@ async def listen_for_messages(consumer):
     to make the service more tolerant of SIGTERM events.
 
     Args:
-        consumer - (AIOKafkaConsumer): kafka consumer for HCCM ingress topic.
+        consumer - (Consumer): kafka consumer for HCCM ingress topic.
 
     Returns:
         None
 
     """
-    LOG.info("Kafka consumer starting...")
-    await consumer.start()
-    LOG.info("Listener started.  Waiting for messages...")
     try:
-        async for msg in consumer:
-            process_complete = False
-            try:
-                process_complete = await process_messages(msg)
-                if process_complete:
-                    await consumer.commit()
-            except (InterfaceError, OperationalError, ReportProcessorDBError) as err:
-                connection.close()
-                LOG.error(f"[listen_for_messages] database error. Seeking to committed. Error: {str(err)}")
-                await asyncio.sleep(Config.RETRY_SECONDS)
-                await consumer.seek_to_committed()
-            except KafkaMsgHandlerError as error:
-                LOG.error(f"[listen_for_messages] internal error. Seeking to committed. Error: {str(error)}")
-                await asyncio.sleep(Config.RETRY_SECONDS)
-                await consumer.seek_to_committed()
-            except ReportProcessorError as error:
-                LOG.error(f"Report processing error: {str(error)}")
-                await consumer.commit()
-
-    except KafkaError as error:
-        LOG.error(f"[listen_for_messages] Kafka error encountered: {type(error).__name__}: {error}", exc_info=True)
-        KAFKA_CONNECTION_ERRORS_COUNTER.inc()
+        offset = msg.offset()
+        partition = msg.partition()
+        LOG.info(f"Processing message offset: {offset} partition: {partition}")
+        process_messages(msg)
+        LOG.debug(f"COMMITTING: message offset: {offset} partition: {partition}")
+        consumer.store_offsets(msg)
```

Comment:
I see `‘enable.auto.offset.store’ must be set to False when using this API` in the docs https://docs.confluent.io/current/clients/confluent-kafka-python/#confluent_kafka.Consumer.store_offsets

Do you know what that's about?